### PR TITLE
Add example SSSD integration using hiera

### PR DESCRIPTION
**Pull request by jeffmccune**

### CODE REVIEW

File: examples/hiera/common.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+## nsswitch configuration required for use with sssd
```
Comment (ghoneycutt) [inline examples/hiera/common.yaml]: Thanks @jeffmccune 

Could you please rename this from common.yaml to RedHat-6.yaml or RedHat-7.yaml, depending on to which platform this corresponds

File: README.md [hunk 2]
```diff
@@ -32,6 +32,11 @@ only), 1.9.3, 2.0.0, 2.1.0 and 2.3.1 (Puppet v4 only).
 
 EL no longer requires the `redhat-lsb` package.
 
+# SSSD
+
+This module works well with sssd.  Please see examples/hiera/common.yaml file
```
Comment (ghoneycutt) [inline README.md]: could you change this to be read which sssd module and provide a link.


```markdown
[sgnl05/sssd](https://github.com/sgnl05/sgnl05-sssd)
```

File: examples/hiera/common.yaml (hunk 1 above)
Comment (jeffmccune) [inline examples/hiera/common.yaml]: Renamed to RedHat-6.yaml in e5700c4

File: README.md (hunk 2 above)
Comment (jeffmccune) [inline README.md]: Hyperlinked to sgnl05/sssd in e5700c4

**Pull request merged**